scheduler/task_priority_cmp.go

```diff
@@ -188,9 +186,9 @@ func (c *byGenerateTasks) compare(t1, t2 task.Task, _ *CmpBasedTaskComparator) (
 // byCommitQueue schedules commit queue merges first
 type byCommitQueue struct{}
 
-func (c *byCommitQueue) name() string { return "task is a commit queue merge" }
+func (c *byCommitQueue) name() string { return "commit queue merge" }
 func (c *byCommitQueue) compare(t1, t2 task.Task, comparator *CmpBasedTaskComparator) (int, string, error) {
-	reason := "higher task is a commit queue merge"
+	reason := "one task is a merge and one is not"
```

Comment:
... so commit queue merge is higher

Reply:
changed to "task that is part of the CQ is higher"